internal/handlers/vulnerabilityreport/trivy_test.go

```diff
@@ -0,0 +1,239 @@
+				{
+					Target: "dbg",
+					Class:  "binary",
+					Type:   "gobinary",
+					Vulnerabilities: []storagev1alpha1.Vulnerability{
+						{
+							CVE:              "CVE-2024-45336",
+							Title:            "Lorem ipsum",
+							PackageName:      "stdlib",
+							PURL:             "pkg:golang/stdlib@v1.23.4",
+							InstalledVersion: "v1.23.4",
+							FixedVersions: []string{
+								"1.22.11",
+								"1.23.5",
+								"1.24.0-rc.2",
+							},
+							DiffID:      "sha256:6d134d3d7e8aa630874f7c4e9db3db48d1895c60f3e5ce73272412404a9b723b",
+							Description: "Lorem ipsum",
+							Severity:    "MEDIUM",
+							References: []string{
+								"https://access.redhat.com/errata/RHSA-2025:3772",
+								"https://access.redhat.com/security/cve/CVE-2024-45336",
+								"https://bugzilla.redhat.com/2341750",
+							},
+							CVSS: map[string]storagev1alpha1.CVSS{
+								"bitnami": {
+									V3Vector: "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
+									V3Score:  "6.1",
+								},
+								"redhat": {
+									V3Vector: "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:N/A:N",
+									V3Score:  "5.9",
+								},
+							},
+							Suppressed: false,
+						},
+						{
+							CVE:              "CVE-2024-45341",
+							Title:            "Lorem ipsum",
+							PackageName:      "stdlib",
+							PURL:             "pkg:golang/stdlib@v1.23.4",
+							InstalledVersion: "v1.23.4",
+							FixedVersions: []string{
+								"1.22.11",
+								"1.23.5",
+								"1.24.0-rc.2",
+							},
+							DiffID:      "sha256:6d134d3d7e8aa630874f7c4e9db3db48d1895c60f3e5ce73272412404a9b723b",
+							Description: "Lorem ipsum",
+							Severity:    "MEDIUM",
+							References: []string{
+								"https://access.redhat.com/errata/RHSA-2025:3772",
+								"https://access.redhat.com/security/cve/CVE-2024-45341",
+								"https://bugzilla.redhat.com/2341750",
+							},
+							CVSS: map[string]storagev1alpha1.CVSS{
+								"bitnami": {
+									V3Vector: "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
+									V3Score:  "6.1",
+								},
+								"redhat": {
+									V3Vector: "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:L/I:L/A:N",
+									V3Score:  "4.2",
+								},
+							},
+							Suppressed: false,
+						},
+					},
+				},
+			},
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			trivyReportFile, err := os.Open(tt.trivyReport)
+			require.NoError(t, err)
+
+			reportData, err := io.ReadAll(trivyReportFile)
+			require.NoError(t, err)
+
+			trivyReportData := &trivyTypes.Report{}
+			err = json.Unmarshal(reportData, trivyReportData)
+			require.NoError(t, err)
+
+			got, err := NewFromTrivyResults(*trivyReportData)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("convertFromTrivyResults() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if !reflect.DeepEqual(got, tt.sbombasticReport) {
+				t.Errorf("convertFromTrivyResults() = %v, want %v", got, tt.sbombasticReport)
+			}
```

Comment:
nit: would be easier to read
```suggestion
			require.Equal(t, got, tt.sbombasticReport)
```